epregressions/runtests.py

```diff
@@ -697,8 +704,7 @@ def process_diffs_for_one_case(self, this_entry, ci_mode=False):
                         EndErrSummary.STATUS_SUCCESS,
                         runtime_case2
                     ))
-                self.my_print("TestMathAndKill Fatal-ed as expected, continuing with no diff checking on it")
-                return this_entry
+                return this_entry, "TestMathAndKill Fatal-ed as expected, continuing with no diff checking on it"
```

Comment:
`self.my_print` is no longer available from this static method, so this function simply returns a string message that can be printed once back in the process pool callback function.